com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.LightLoop.cs

```diff
@@ -159,7 +160,7 @@ static void ClearLightLists(BuildGPULightListPassData data, CommandBuffer cmd)
                 // Also, we clear all the lists and to be resilient to changes in pipeline.
                 if (data.runBigTilePrepass)
                     ClearLightList(data, cmd, data.output.bigTileLightList);
-                if (data.lightList != null) // This can happen for probe volume light list build where we only generate clusters.
+                if (data.canClearLightList) // This can happen for probe volume light list build where we only generate clusters.
```

Comment:
Note mostly for myself: we don't have a probe volume light list build, so is this needed? If so, update comment.

Reply:
Commend updated